doc/source/workflow/install.md

```diff
@@ -77,6 +77,18 @@ helm install seldon-core seldon-core-operator \
     --set certManager.enabled=true
 ```
 
+### Install behind a proxy
```

Comment:
This is really good - given how specific this is may be best to append this section at the end of the doc

Reply:
@axsaucedo updated